bamboo/bootstrap-unit-tests.sh

```diff
@@ -14,6 +14,12 @@ docker_command="docker exec -t ${container_id}-build_env-1 /bin/bash -c"
 
 docker ps -a
 
+echo 'Removing running docker containers...'
```

Comment:
This is an unrelated cleanup that is now a separate PR https://github.com/nasa/cumulus/pull/3586 and should probably be removed in favor of #3586